source/admin_manual/submission_server.rst

```diff
@@ -79,8 +79,7 @@ configuring Dovecot's URLAUTH support can be found at
 
 The following settings apply to the Submission service:
 
-submission_logout_format = in=%i out=%o
-***************************************
+**submission_logout_format = in=%i out=%o**
```

Comment:
No need for `source: ` prefix in title, since everything goes under source. Also other commits could have some kind of a prefix (admin_manual: at least).